src/de/xite/scoreboard/modules/board/ScoreboardManager.java

```diff
@@ -90,7 +90,7 @@ private void importScores(YamlConfiguration cfg) {
 			try {
 				int id = Integer.parseInt(s);
 				
-				if(cfg.getStringList(id+".scores") != null && !cfg.getStringList(id+".scores").isEmpty()) {
+				if(!cfg.getStringList(id+".scores").isEmpty()) {

```

Comment:
Even if it should not be the case, the list could be null for whatever reason.
If the array is null and you call isEmpty, it will throw an error.

Reply:
```suggestion
				if(cfg.getStringList(id+".scores") != null && !cfg.getStringList(id+".scores").isEmpty()) {

```

Reply:
Just saw that it will actually return an empty list if the list does not exists.
However it might be better to define the list above the if statement and directly access the list..
```java
List<String> list = cfg.getStringList(id+".scores");
if(!list.isEmpty()) {
	int speed = cfg.getInt(id+".speed");
```

Reply:
You're pretty right. My bad! The updates has been commited.

Reply:
Sorry for confusion. Actually your original change was correct, the null check isn't needed. Initializing the list above the if statement is just further improvement.

Please remove the null check again and  leave the rest (list initialization) as is.

Reply:
The null check is needed for the older versions, i think. Nah? Spigot's older versions doesn't have the @NotNull and @Nullable annotations from jetbrains.

Reply:
I don't know exactly but that could be the case. Then just leave everything as is now. Rather a check more than less. I will merge it tomorrow 👍

Reply:
Amazing! I'll contact you in Discord tomorrow to discuss some more changes that i'm planning to do, if you don't matter.